torch/ao/quantization/_pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -0,0 +1,205 @@
+import torch
+import torch.nn.functional as F
+import copy
+import functools
+import itertools
+from .quantizer import (
+    OperatorConfig,
+    OperatorPatternType,
+    QuantizationConfig,
+    QuantizationSpec,
+    Quantizer,
+    QuantizationAnnotation,
+)
+from torch.ao.quantization._pt2e.quantizer.utils import (
+    get_input_act_qspec,
+    get_output_act_qspec,
+    get_weight_qspec,
+    get_bias_qspec,
+)
+from .qnnpack_quantizer import (
+    _is_annotated,
+)
+from torch.ao.quantization.observer import (
+    HistogramObserver,
+    PlaceholderObserver,
+    PerChannelMinMaxObserver,
+)
+from torch.ao.quantization.qconfig import _ObserverOrFakeQuantizeConstructor
+from typing import List, Dict, Optional, Set, Any
+from torch.fx import Node
+from torch.fx.passes.utils.source_matcher_utils import get_source_partitions
+
+__all__ = [
+    "X86InductorQuantizer",
+    "get_default_x86_inductor_quantization_config",
+]
+
+def supported_quantized_operators() -> Dict[str, List[OperatorPatternType]]:
```

Comment:
add TODO comment to add more supported operators?

Reply:
Thanks for the suggestion and added.